objects/Oracle.js

```diff
@@ -639,12 +640,12 @@ module.exports = {
         Promise.all(lookUpUsers).then(results => {
             for (let i = 0; i < results.length; i++){
                 if (results[i] !== "Can't find deck"){
-                    let calculatedWinrate = Math.round(results[i][0][1]/(results[i][0][1]+results[i][0][2])*100);
-                    let elo = (30*(results[i][0][1])) - (10*(results[i][0][2])) + 1000;
+                    let calculatedWinrate = Math.round(results[i][0][1]/(results[i][0][1]+results[i][0][3])*100);
                     let username = results[i][0][0];
-                    let gamesPlayed = (results[i][0][1] + results[i][0][2]);
+                    let gamesPlayed = (results[i][0][1] + results[i][0][3]);
+                    elo = (results[i][0][2]) - (results[i][0][4]) + 1000;
```

Comment:
I notice you created a startingElo that holds this 1000 number.  Can we add that to these areas?  One day we want to make that a variable as well.